app/facades/photo_facade.rb

```diff
@@ -1,9 +1,10 @@
 class PhotoFacade
   def self.photos(word)
     @photo = Unsplash::Photo.search("#{word}")
-  end
-
-  def self.single_photo
-    @photo.sample.to_json
+    pr = JSON.parse(@photo.sample.to_json, symbolize_names: true)
```

Comment:
I believe a clearer variable name would be empathetic here (to replace "pr")

Reply:
yes, will update to reflect

Reply:
Updated, will push changes when entire punch list is complete